packages/grafana-data/src/transformations/transformers.ts

```diff
@@ -23,4 +24,5 @@ export const standardTransformers = {
   calculateFieldTransformer,
   seriesToColumnsTransformer,
   renameFieldsTransformer,
+  labelsAsColumnsTransformer,
```

Comment:
can we rename it to labelsToFieldsTransformer?

Reply:
Rename done!